recipe/meta.yaml

```
@@ -24,7 +24,7 @@ requirements:
   run:
     - python >={{ python_min }},<{{ python_max }}
     - anytree >=2.8.0
-    - casadi >=3.6.6
+    - casadi >=3.6.7
```

```suggestion
    - casadi >=3.6.7
    - posthog >=3.6.5
```

24.11 now has telemetry using `posthog`.